GAE/war/WEB-INF/web.xml

```diff
@@ -480,6 +480,14 @@
 		<servlet-name>RemoteApi</servlet-name>
 		<url-pattern>/remote_api</url-pattern>
```

Comment:
Taking the deleted class in to account, should we then also only add an extra `<url-pattern></url-pattern>` here for tracing and remove the servlet definition below?